src/Model/Mail.php

```diff
@@ -46,6 +46,8 @@ public static function insert($msg)
 			$msg['guid'] = Item::guidFromUri($msg['uri'], $host);
 		}
 
+		$msg['created'] = (isset($msg['created']) ? DateTimeFormat::utc($msg['created']) : DateTimeFormat::utcNow());
```

Comment:
You can even use `!empty()` since all falsy values are treated as `now`.
```suggestion
		$msg['created'] = (!empty($msg['created']) ? DateTimeFormat::utc($msg['created']) : DateTimeFormat::utcNow());
```